include/ensmallen_bits/problems/maf/maf4_function.hpp

```diff
@@ -0,0 +1,225 @@
+/**
+ * @file maf4_function.hpp
+ * @author Satyam Shukla
+ *
+ * Implementation of the fourth Maf test.
+ *
+ * ensmallen is free software; you may redistribute it and/or modify it under
+ * the terms of the 3-clause BSD license.  You should have received a copy of
+ * the 3-clause BSD license along with ensmallen.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+
+#ifndef ENSMALLEN_PROBLEMS_MAF_THREE_FUNCTION_HPP
+#define ENSMALLEN_PROBLEMS_MAF_THREE_FUNCTION_HPP
```

Comment:
This is the same as in `maf4_function.hpp`, so this file won't ever actually be read.  Maybe can we make sure before we merge this PR that these functions are used in some test somewhere?

Reply:
Sure will post some results this weekend with NSGA2 and AGEMOEA The consistent ones can be turned into tests

Reply:
Great!